test/cli.test.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+const t = require('tap')
+const split = require('split2')
+const path = require('path')
+const childProcess = require('child_process')
+const helper = require('./helper')
+
+const lines = [
+  /Running 1s test @ .*$/,
+  /10 connections.*$/,
+  /$/,
+  /Stat.*Avg.*Stdev.*Max.*$/,
+  /Latency \(ms\).*$/,
+  /Req\/Sec.*$/,
+  /Bytes\/Sec.*$/,
+  /$/,
+  /\d+k requests in 1s, .* MB read$/
```

Comment:
I know it _should_ be 1s, it _could_ be longer than 1s... could we change it to a selection group of [1|2]? any longer is a fail for sure :)